modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json

```diff
@@ -42,6 +42,10 @@
 		"cssVariableMapping": "color-brand-primary-darken-5",
 		"value": "#00256C"
 	},
+	"colorNeutral2": {
+		"cssVariableMapping": "color-neutral-2",
```

Comment:
You could squash the two commits that add color to frontend tokens.  No need for them to be separate commits.